.github/workflows/ci.yaml

```diff
@@ -0,0 +1,281 @@
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' || github.ref_name == 'master' }}
+    needs:
+      - test
+      - test-hsm
+      - test-e2e
+      - docs-build
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ory/ci/changelog@master
+        with:
+          token: ${{ secrets.ORY_BOT_PAT }}
+
+  sdk-generate:
+    name: Generate SDKs
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' || github.ref_name != 'master' }}
+    needs:
+      - test
+      - test-hsm
+      - test-e2e
+    steps:
+      - uses: ory/ci/sdk/generate@master
+        with:
+          token: ${{ secrets.ORY_BOT_PAT }}
+          app-name: Ory_Hydra
+          swag-spec-ignore: internal/httpclient gopkg.in/square/go-jose.v2
+          swag-spec-location: spec/api.json
+
+  sdk-release:
+    name: Release SDKs
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - test
+      - test-hsm
+      - sdk-generate
+      - docs-build
+      - release
+    steps:
+      - uses: ory/ci/sdk/release@master
+        with:
+          token: ${{ secrets.ORY_BOT_PAT }}
+          app-name: Ory_Hydra
+          swag-spec-ignore: internal/httpclient gopkg.in/square/go-jose.v2
+          swag-spec-location: spec/api.json
+
+  release:
+    name: Generate release
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - test
+      - test-hsm
+      - test-e2e
+      - changelog
+      - docs-build
+    steps:
+      - uses: ory/ci/releaser@master
+        with:
+          token: ${{ secrets.ORY_BOT_PAT }}
+          goreleaser_key: ${{ secrets.GORELEASER_KEY }}
+          cosign_pwd: ${{ secrets.COSIGN_PWD }}
+          docker_username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker_password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+  render-version-schema:
+    name: Render version schema
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - release
+    steps:
+      - uses: ory/ci/releaser/render-version-schema@master
+        with:
+          schema-path: spec/config.json
+
+  newsletter-draft:
+    name: Draft newsletter
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - release
+    steps:
+      - uses: ory/ci/newsletter@master
+        with:
+          mailchimp_list_id: 058a056618
+          mailchmip_segment_id: 11398953
+          mailchimp_api_key: ${{ secrets.MAILCHIMP_API_KEY }}
+          draft: 'true'
+          ssh_key: ${{ secrets.ORY_BOT_SSH_KEY }}
+
+  slack-approval-notification:
+    name: Pending approval Slack notification
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - newsletter-draft
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+    steps:
+      - uses: slackapi/slack-github-action@v1.16.0
+        with:
+          payload: |
```

Comment:
Given that we need to have this for all projects, can this be moved to its own action so we do not have to write the config JSON for slack in all actions?

Reply:
Good call! Will do.